src/shared/farcaster/api.ts

```diff
@@ -0,0 +1,9 @@
+export const getUserFid = (username: string) => {
+  return fetch(
+    `https://fnames.farcaster.xyz/transfers/current?name=${username}`,
+  );
+};
+
+export const getAccountByFid = (fid: number) => {
+  return fetch(`https://api.idriss.xyz/get-connected-addresses?fid=${fid}`);
+};
```

Comment:
@SolutionsEngineer WDYT about making this kind of `api` files instead of calling fetch directly in command handlers?

Previously I thought command handlers will only be responsible for calling api's but in reality they often do some extra logic.

Imho we could have these descriptive "api methods" in `api.ts` file in each module that defines some requests and probably we could have schema validation inside so we have type-safety and good DX for return type of these api methods.

Reply:
Agree, moving the schema validation to the API methods would also reduce some code in the command files, making them easier to read if additional logic is added. I'm only afraid that the API methods file might grow too large, but since an api.ts file would be created for each module, I don’t think it will get messy